.github/dependabot.yml

```diff
@@ -6,11 +6,31 @@
 version: 2
 updates:
   - package-ecosystem: "npm" # See documentation for possible values
-    directory: "/Website" # Location of package manifests
+    directory: "/Website/Website" # Location of package.json
```

Comment:
Any reason why the other NPM packages are excluded from the list? Following are the missing packages
1. /Website/Packages/datax-jobs
2. /Website/Packages/datax-metrics
3. /Website/Packages/datax-pipeline
4. /Website/Packages/datax-query

Reply:
added